packages/core/src/plugins/meta/discogs.ts

```diff
@@ -105,6 +114,11 @@ class DiscogsMetaProvider extends MetaProvider {
   }
 
   discogsTrackToGeneric(discogsTrack: DiscogsTrack, artist: string): Track {
+
+    if (discogsTrack.type_ !== 'track') {
```

Comment:
You can just filter out non-tracks instead of filling the list with dummy entries and then filtering them out later.